db/seeds.rb

```diff
@@ -1,5 +1,9 @@
-
 # Setup user/dev for example site
+developer = FactoryGirl.build :developer
```

Comment:
Ah yeh we haven't updated the seeds in a while. There is also another expected default developer, with the api key found under secrets['coposition_api_key'] so probably worth adding that to the seeds as well.